js/foam/graphics/ActionButtonCView.js

```diff
@@ -210,7 +210,7 @@ CLASS({
 
       // Pressing space when has focus causes a synthetic press
       this.$.addEventListener('keypress', function(e) {
-        if ( ( e.charCode == 32 || e.charCode == '13')&& ! ( e.altKey || e.ctrlKey || e.shiftKey ) ) {
```

Comment:
Reserving the enter key for the equals operator makes more sense then my previous decision to allow it to trigger buttons